### DISCUSSION TITLE
VBET 3.0.7: Questions about the Relative Likelihood Raster & VB extent mapping

### DISCUSSION THREAD

**Discussion by shelbysawyer** [Riverscapes/riverscapes-tools#791] [VBET]
While working in a 3.0.7 project ([Upper Cheat River, 0502000405](https://warehouse.riverscapes.net/p/3d87f2d7-fbb7-4f2c-bcd3-bdcc0a6ad622)), I got confused in a couple places about how the relative likelihood raster behaves and how VB extent is generated. @jtgilbert @joewheaton **I need help understanding why it looks the way it does so that I can document it properly.** I have a few questions scattered below:

These seemingly ownerless bits of valley bottom were what first got my attention:
 
![extra](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/da61fb37-542f-42de-a976-089205fd16fb)

It turns out they belong to this branch of the mainstem, which has a different levelpath from the adjacent mainstem reach:
![branch](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/b2fac775-6b28-445e-8adf-6100d5f2f176)

Here's the DEM and VB extents mapped and imagery for some extra context:
![dem](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/d693e2e2-f61b-4d2a-85c6-3ada3b41f7cf)
![imagery](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/4d95009e-911b-458e-86c1-cadfeebd5259)

What's confusing to me is that the branch falls completely within the valley bottom of the mainstem levelpath. I thought we were removing valley bottoms of levelpaths that fell within the valley bottoms of larger levelpaths. Additionally, the HAND evidence shows the branch as being 0m higher than the mainstem and slope being very very low:
![hand](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/5f332e16-9c63-4daa-9c1c-a329e96aea55)
![slope](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/97ba7acf-0a68-40b1-819d-91ab2516157e)

And there are 0.995 (estimated active channel) values falling outside the valley bottom, which doesn't make sense to me, because the original seemingly-ownerless bits of valley bottom are lower likelihood and still propted the drawing of valley bottom extent:
![extra3](https://github.com/Riverscapes/riverscapes-tools/assets/73311751/4581eaa6-ff9d-46e1-a46f-5deb69bfa975)

I'm guessing the 0.995 values that are falling outside of the mainstem valley bottom are being generated from the branch. In my mind, it would make sense for the branch to also be generated as 0.995 within the mainstem valley bottom-- @jtgilbert is it not just because we have slope evidence decreasing with distance?

**Comment by jtgilbert** [comment]
re: those 'ownerless' cells, VBET does exclude level paths within the mainstem VB, BUT it has to be entirely contained. If you look closely the channel area for that side channel is just barely outside the mainstem VB at a few cell corners, so it remains included. When it's VB is delineated it is just barely (1 cell) beyond what got calculated for the mainstem, and because those cells intersect the channel area polygon they are retained.

I think the 1 values outside the VB are a result of how the composites are assembled. That's a larger issue I'd like to dive into at some point...
